### DISCUSSION TITLE
Question about ports and user restrictions

### DISCUSSION THREAD

**Discussion by BazZziliuS** [pterodactyl/panel#4695] [Support] [answered]
Hello, I have two questions, I hope you will help me.

1. How to add a local IP:PORT for a pterodactyl?
I am running a Minecraft server and it cannot connect to the database due to the fact that it cannot connect to the local ip:port.

2. Is there any way to restrict access to certain server files? For example, I add a new user to the server and I don't want him to be able to download logs

**Comment by parkervcp** [comment, marked as answer]
You cannot use locahost. Each server runs in an isolated docker container. The closest you can get is using the `172.18.0.1` address with the port on it.


See the guide for setting up a db for pterodactyl nodes - https://pterodactyl.io/tutorials/mysql_setup.html#creating-a-database-host-for-nodes
